frontend/src/metabase/collections/components/CollectionSidebar/Bookmarks/Bookmarks.tsx

```diff
@@ -21,23 +23,47 @@ import {
 
 import { Bookmark, BookmarkableEntities, Bookmarks } from "metabase-types/api";
 
-interface LabelProps {
-  name: string;
-  display?: string;
-  type: BookmarkableEntities;
+interface BookmarkProps {
+  bookmark: Bookmark;
 }
 
 interface CollectionSidebarBookmarksProps {
   bookmarks: Bookmarks;
   deleteBookmark: (id: string, type: string) => void;
 }
 
-const Label = ({ display, name, type }: LabelProps) => {
-  const iconName = getIcon(display, type);
+interface IconProps {
+  name: string;
+  tooltip?: string;
+  isOfficial?: boolean;
+}
+
+const BookmarkIcon = ({ bookmark }: BookmarkProps) => {
+  const icon = BookmarkEntity.objectSelectors.getIcon(bookmark);
+  const isCollection = bookmark.type === "collection";
+  const isRegularCollection =
+    isCollection && PLUGIN_COLLECTIONS.isRegularCollection(bookmark);
+  const isOfficial = isCollection && !isRegularCollection;
+
+  const iconColor = isOfficial ? color("warning") : color("brand");
```

Comment:
Please let's not hardcode the official collection's color here and just use the color that the collection's `getIcon` returned above

Reply:
@kulyk In this case, the icons for Questions would be dark gray, and we want them blue.

<img width="441" alt="image" src="https://user-images.githubusercontent.com/380816/160672510-0ae3e0cc-14d6-4090-9827-43102c934ce3.png">

Reply:
Can we remove the grey color from question's `getIcon` function and set it in the components instead? I believe it shouldn't be there at all

Reply:
We can conditionally only overrule the color if it's not an official collection.

No lack of possibilities here, whatever you prefer I'm fine with.

Reply:
Ah sure. I thought we were gonna do it in a follow-up PR. 

No idea how many of these we have anyway.
I'll do it here.